params/licenses.go

```diff
@@ -0,0 +1,37 @@
+package params
```

Comment:
What do you think about renaming this package to something more unambiguous, like `licenses` or `versioning` or something like that?

Reply:
It's possible, but there are not so much codes in each package as you can see

Reply:
Yes, but the package name is ambiguous. Params for what? What kind of params?

I just think we can have better naming of the package, instead of `params`

Reply:
Fixed 9de25d8a72538a7672794d0a446e31406e8d3fac